test/k8s-integration/main.go

```diff
@@ -39,6 +39,8 @@ var (
 	kubeVersion      = flag.String("kube-version", "master", "version of Kubernetes to download and use")
 	kubeFeatureGates = flag.String("kube-feature-gates", "", "feature gates to set on new kubernetes cluster")
 	localK8sDir      = flag.String("local-k8s-dir", "", "local kubernetes/kubernetes directory to run e2e tests from")
+	deploymentStrat  = flag.String("deployment-strategy", "gce", "choose between deploying on gce or gke")
+	gkeClusterVer    = flag.String("gke-cluster-version", "latest", "version of Kubernetes master and node for gke")
```

Comment:
Can we reuse kube-version?

Reply:
Its a little hard, because "master" is not an option for GKE. And also the set of possible options for GCE and GKE are different (i.e. a GKE option can be 1.13.6-gke.6). Because there is essentially no overlap in the version names and also the default option, I felt it would be less confusing to use a new flag, but thoughts?

Reply:
Also, the kube-version is used to pull the repo to build tests, and repo versioning is not the same as gke versioning.

Reply:
Ah I see, I forgot we still need to download kubernetes to get the test suite even for GKE runs.  This makes sense then.